src/components/network-chart/NetworkChartRow.js

```diff
@@ -281,14 +280,20 @@ class NetworkChartRow extends React.PureComponent<NetworkChartRowProps, State> {
         </div>
         <div
           className="networkChartRowItemBar"
-          style={markerStyle}
+          style={{ width: markerWidth, left: startPosition }}
           onMouseEnter={this._hoverIn}
           onMouseLeave={this._hoverOut}
         >
-          <NetworkChartRowBar marker={marker} networkPayload={networkPayload} />
+          <NetworkChartRowBar
+            marker={marker}
+            networkPayload={networkPayload}
+            markerWidth={markerWidth}
+          />
         </div>
         {this.state.hovered ? (
-          <Tooltip mouseX={this.state.pageX} mouseY={this.state.pageY}>
+          // This magic value "5" avoids the tooltip of being too close of the
+          // row, especially when we mouseEnter the row from the top edge.
+          <Tooltip mouseX={this.state.pageX} mouseY={this.state.pageY + 5}>
```

Comment:
This wasn't strictly necessary here but this felt better when I used it. In the future I'd like to use the actual marker position rather than the mouse position, and display the tooltip when we hover any location of the row, not the specific marker (as sometimes the marker can be very small). But this seemed like too much work for this PR.